vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -1232,6 +1239,8 @@ def select_experts(
         expert_load_view: Optional[torch.Tensor] = None,
         logical_to_physical_map: Optional[torch.Tensor] = None,
         logical_replica_count: Optional[torch.Tensor] = None,
+        num_fused_shared_experts: int = 0,
+        routed_scaling_factor: Optional[float] = None,
```

Comment:
Can we assert that both of these are 0/None for now since they aren't hooked up anywhere?